backend/papersfeed/utils/users/utils.py

```diff
@@ -144,8 +144,11 @@ def update_user(args):
     # Password
     password = args[constants.PASSWORD] if constants.PASSWORD in args else None
 
+    # User Photo Index
+    photo_index = args[constants.PHOTO_INDEX] if constants.PHOTO_INDEX in args else None
+
     # Update Descrpition
-    if description:
+    if description is not None: #deleting description also should be supported
```

Comment:
Good debugging!

Reply:
I think Python's `if {object}:` is sometimes dangerous, because it is false if the {object} is list or string whose length is 0.